src/ext/Bal/wixext/WixToolset.Bal.wixext.nuspec

```diff
@@ -20,9 +20,5 @@
     <file src="$projectFolder$\..\..\..\internal\images\wix-white-bg.png" />
 
     <file src="$id$.dll" target="tools" />
-
-    <file src="..\x86\*.pdb" target="pdbs\x86" />
-    <file src="..\x64\*.pdb" target="pdbs\x64" />
-    <file src="..\ARM64\*.pdb" target="pdbs\ARM64" />
```

Comment:
I think this means we can remove the .nuspec and use the builtin pack functionality?

Reply:
Maybe.